plugins/modules/ecs_ecr.py

```diff
@@ -248,17 +272,21 @@ def get_repository_policy(self, registry_id, name):
         except is_boto3_error_code(['RepositoryNotFoundException', 'RepositoryPolicyNotFoundException']):
             return None
 
-    def create_repository(self, registry_id, name, image_tag_mutability):
+    def create_repository(self, registry_id, name, image_tag_mutability, encryption_configuration):
         if registry_id:
             default_registry_id = self.sts.get_caller_identity().get('Account')
             if registry_id != default_registry_id:
                 raise Exception('Cannot create repository in registry {0}.'
                                 'Would be created in {1} instead.'.format(registry_id, default_registry_id))
 
+        if encryption_configuration is None:
+            encryption_configuration = dict(encryptionType='AES256')
```

Comment:
```suggestion
        if encryption_configuration is None:
            encryption_configuration = dict(encryptionType='AES256')
```

This makes it impossible to create a none-encrypted ECR.

Reply:
ah wait. AES256 is the default